warehouse/query-core/src/main/java/datawave/core/iterators/DatawaveFieldIndexCachingIteratorJexl.java

```diff
@@ -1392,15 +1366,16 @@ protected void setupRowBasedHdfsBackedSet(String row) throws IOException {
 
         try {
             // for each of the ivarator cache dirs
```

Comment:
Nothing really happens in this part unless this.allowDirReuse == false, so check that first